src/main/webapp/js/instructorStudentEdit.js

```diff
@@ -0,0 +1,22 @@
+$(document).ready(function() {
+    readyStudentEditPage();
```

Comment:
@Sumei1009 this should be `readyInstructorStudentEditPage()` as defined down there. This mismatch caused a UI test to fail. I've fixed it while merging so this is no longer a problem, but do be very careful next time.